Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py

```diff
@@ -125,7 +127,15 @@
         'Custom' : 'SLHCUpgradeSimulations/Configuration/combinedCustoms.cust_2023tilted',
         'Era' : 'Phase2C2_timing',
         'ScenToRun' : ['GenSimHLBeamSpotFull','DigiFull','RecoFullGlobal', 'HARVESTFullGlobal'],
+    },
+    '2023D6' : {
+        'Geom' : 'Extended2023D6',
+        'GT' : 'auto:run2_mc',
+        'Custom' : 'SLHCUpgradeSimulations/Configuration/combinedCustoms.cust_2023tilted',
+        'Era' : 'Phase2C2',
```

Comment:
Ah, this should be ```'Era' : 'Phase2C1'```. Given that D6 is defined with C1 as the nominal calorimeter.